src/vendor/GLTFLoader.js

```diff
@@ -14,6 +15,7 @@ THREE.GLTFLoader = ( function () {
 
 		this.manager = ( manager !== undefined ) ? manager : THREE.DefaultLoadingManager;
 		this.dracoLoader = null;
+		this.preferredTechnique = null;
```

Comment:
Any reason not to make this a constructor parameter? (Or maybe add an options map to the constructor instead)

Reply:
I'm just following what the Draco loader is doing. I'll submit a PR to Three.js and see what they think.